internal/resources/cluster/helper_test.go

```diff
@@ -60,6 +60,7 @@ func withTKGsCluster() testAcceptanceOption {
 func withTKGmVsphereCluster() testAcceptanceOption {
 	return func(config *testAcceptanceConfig) {
 		config.ManagementClusterName = os.Getenv("MANAGEMENT_CLUSTER")
+		config.ProvisionerName = os.Getenv("PROVISIONER_NAME")
```

Comment:
Why have we added this as an environment variable? We can just use value of Provisioner as `default` and not take the pain of setting it every time while running the acceptance test. As it will be the maintainers' responsibility to run them every time. Just a suggestion.

Reply:
What if we have different provisioners? Hence, it is better to keep it as an environment variable. We had a bug where we were setting the provisioner name to attach in case where we were only running the TKGm acceptance test, this fixes it.